### ISSUE TITLE
HELP CAN'T SEND MY ETH MAINET BALANCE

### ISSUE THREAD

**Issue by gwap0ako025** [walleth/walleth#452]
Please help me out everytime that i send  some of my balance to my coinbase  using this WALLeth app it won't allow me   
![Screenshot_2020-04-24-03-10-15-16_0df7059f534a28aebebcd10014a2a043](https://user-images.githubusercontent.com/62929119/80144317-dae79500-85e0-11ea-9ded-fd32c5ae9bcf.png)

I just downloaded this app from google play and send all my eth from walleth and it won't allow me to send  :( help

**Comment by gwap0ako025** [comment]
Why i can't send some or my funds  ?  What is wrong about this app

**Comment by ligi** [comment]
what is happening when you try to send funds?

**Comment by gwap0ako025** [comment]
It's asking me to set it up manually when i set the numbers  it won't allow
me to send my funds. I'm not a developer i'm just a user  just want to save
all his earnings please help

**Comment by ligi** [comment]
> "It's asking me to set it up manually when i set the numbers "

you mean an error about estimating gas prices?

what does the error say when you try to send funds?

I am about to help you - please stay calm.

**Comment by gwap0ako025** [comment]
YES AND ALSO WHEN I USE THE WEB 3 TRIED TO CONNECT IT IN MYCRYPTO OR
MYETHETWALLETH STILL IT WON'T ALLOW ME TO  CONFIRM THE TRANSACTION  TO
CONFIRM IT.

**Comment by ligi** [comment]
there is no "sign in" - can you please tell me what error message you get?

if you are in a hurry what you can also do is to export your key and import it into another wallet

**Comment by gwap0ako025** [comment]
Will you please help me transfer all myfunds to my coinbase wallet or
blockhain wallet

**Comment by gwap0ako025** [comment]
It won't allow me to continue on the next part .

**Comment by ligi** [comment]
there is an export key option in the edit account section - there you can export the key

**Comment by ligi** [comment]
> YES AND ALSO WHEN I USE THE WEB 3 TRIED TO CONNECT IT IN MYCRYPTO OR
MYETHETWALLETH STILL IT WON'T ALLOW ME TO  CONFIRM THE TRANSACTION  TO
CONFIRM IT.

I do not understand what you are saying there

**Comment by ligi** [comment]
could you export your account?

**Comment by gwap0ako025** [comment]
I tried using my myetherwallet but when i'm importimg the json file from a
other address it won't allow me  .

**Comment by ligi** [comment]
please be more precise - what do you mean by "from a other address" and "won't allow me"?

**Comment by gwap0ako025** [comment]
When import it the jsonfile that i downloaded from myetherwallet it won't
allow me to access ut from another web3 wallets  like meta masked still
asking for the jsonfile password  i put the password that i always used and
still its telling me password is wrong.

**Comment by ligi** [comment]
but what does this have to do with WallETH?

**Comment by gwap0ako025** [comment]
Regards on the sign in message and verifykng the transaction the web3
wallet always asked me to confirm transactions in WALLeth APP

**Comment by ligi** [comment]
where did you create the account in question? Really not sure what you are talking about

**Comment by gwap0ako025** [comment]
The address i can't remember if thats from atomic or from changelly or
blockmove.io coz i created lots of wallet address just to fit in
etherscan.io  then i was got excited whe  i'm getting 30 eth in my address
thats why i haven't read the description nor even save prvt keys

**Comment by gwap0ako025** [comment]
Coz for me all wallets are the same i don't know that web3 wallets or erc20
tokens ate difficult to handle  :(

**Comment by ligi** [comment]
Sorry I really do not know how to help you here.

**Comment by lee-tts** [comment]
I have the same issue and can perhaps provide more ... prosaic responses. 

I am using the F-Droid Walleth version 0.48.2 (I do not have Play store on this phone)

I sent a test tx to this wallet and now cannot get it to send back out. I have tried twice to send the test amount to a metamask wallet, and the txs are sitting in the main screen, but the tx hash does not appear in etherscan. So they don't seem to have been broadcast to the network. 

I have also noticed fee/gas estimation fails due to an error and also the Open in Blockexplorer option in the 3-dot menu yields a 503 error on the destination page.

Any help much appreciated!

**Comment by djyotta** [comment]
I'm on 0.48.4 from the Google Play store and also can not send transaction. Remains pending for days (but without the JSON errors I was getting on 0.48.2) and transaction doesn't show in etherscan.

Let me know what I can do to help debug.

**Comment by djyotta** [comment]
Ok, in my case, Gas price was too low. I did follow recommendation from ethgasstation.info but after importing my wallet into another app and seeing the lowest Gas price offered there, I bumped it up in walleth and the transaction appears on etherscan now.
Gas price is 65000000000 in walleth. Before I was using 10000000000. I think that's 65 and 10 Gwei respectively... not sure. Gas limit was always 21000

**Comment by lee-tts** [comment]
Sounds like I need to wait for the F-Droid version to update then.

**Comment by gwap0ako025** [comment]
What wallet you send it  ?

**Comment by djyotta** [comment]
@gwap0ako025 I may need  little more context. I presume you mean which features the recieving wallet had?

In any case, I'm not entirely sure. But I can find out for you if that would help.

**Comment by gwap0ako025** [comment]
I don't remember

**Comment by gwap0ako025** [comment]
0xf9c18f2d1ea93e9a36507f238cffc6f0ad7d6eb3

That's my wallet

**Comment by lkblkb** [comment]
@gwap0ako025  You need to give more information. 

That is your address. Not your wallet :)
